addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/ManualHttpRequestEditorPanel.java

```diff
@@ -553,6 +553,7 @@ public void changeView(int newView) {
 
                 this.validate();
                 this.repaint();
+                this.saveConfig();
```

Comment:
This method saves more than just the layout, e.g. the current view of a panel. Doing just here leads to inconsistencies, in that sometimes the current view is the last chosen and other times it isn't. If we want all changes (of the last changed panel) to be applied to new panels it might be better to save the configs before creating a new panel instead. Otherwise this should just save the layout.

Reply:
Okay will dig further.